packages/server/src/api/roster/roster-entity.ts

```diff
@@ -41,6 +40,12 @@ export abstract class RosterEntity extends BaseEntity {
   @PrimaryGeneratedColumn()
   id!: number;
 
+  @ManyToOne(() => Org, org => org.id, {
+    nullable: false,
+    onDelete: 'CASCADE',
+  })
+  org!: Org;
```

Comment:
Since this is a relation without `eager: true` it's best to type it as possibly undefined in case roster entries are queried without it.

```suggestion
  org?: Org;
```